src/main/flight/mixer.c

```diff
@@ -345,6 +344,59 @@ static void applyFlipOverAfterCrashModeToMotors(void)
         }
     }
 }
+#ifdef USE_RPM_LIMITER
+static float getAverageRPM(void) // Placeholder function
+{
+    float averageRPM = 0;
+    for (int i = 0; i < getMotorCount(); i++) {
+        averageRPM += getDshotTelemetry(i);
+    }
+    return (averageRPM / (getMotorCount() * motorConfig()->motorPoleCount)) * 20;
+}
```

Comment:
```suggestion
```

You can get rid of this because we already have a function that calculates the average RPM. See
https://github.com/betaflight/betaflight/blob/master/src/main/drivers/dshot.c#L326-L329